app/celery/tasks.py

```diff
@@ -249,9 +258,18 @@ def __sending_limits_for_job_exceeded(service, job: Job, job_id):
     return False
 
 
+# TODO: replace with real thing
+class RedisQueue:
+    def __init__(self, type):
+        pass
+
+    def acknowledge(self, receipt):
+        pass
+
+
 @notify_celery.task(bind=True, name="save-smss", max_retries=5, default_retry_delay=300)
 @statsd(namespace="tasks")
-def save_smss(self, service_id: str, encrypted_notifications: List[Any]):
+def save_smss(self, service_id: str, encrypted_notifications: List[Any], receipt: str):
```

Comment:
can we update the doc string for receipt and how it is connected to the post vs bulk?